tasks/dist.js

```diff
@@ -39,16 +40,21 @@ module.exports = function(gulp, config) {
 
 	});
 
-	var buildTasks = ['build:dist:scripts'];
+	var buildTasks = ['build:dist:scripts', 'build:dist:css'];
 
-	if (config.component.less) {
-		gulp.task('build:dist:css', ['clean:dist'], function() {
-			return gulp.src(config.component.less)
-				.pipe(less())
-				.pipe(gulp.dest('dist'));
-		});
-		buildTasks.push('build:dist:css');
-	}
+
+	gulp.task('build:dist:css', ['clean:dist'], function() {
+
+		return gulp.src('./' + config.component.src + '/' + config.component.name + '.less')
+			.pipe(gulp.dest('dist'))
```

Comment:
I like your change to add `.min` via `gulp-minify-css`.

But I'm not sure that there's value in copying the .less into the `/dist` folder when it can be included directly from `/src`...?

Everything else in `/dist` is transformed.

Importantly, this should only be included if `config.component.less` is set, which you have excluded. The line below, `buildTasks.push();` is useless.